src/main/java/uk/gov/hmcts/cmc/claimstore/exceptions/ClaimantResponseAlreadySubmittedException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.cmc.claimstore.exceptions;
+
+public class ClaimantResponseAlreadySubmittedException extends ConflictException {
+    public ClaimantResponseAlreadySubmittedException(long claimId) {
```

Comment:
will prefer not to use DB generated id here, can be UUID or similar which will work for CCD as well

Reply:
I can change it, but
1. Its consistent with defendant response
2. it would work just fine on ccd